alfresco-transform-imagemagick/alfresco-transform-imagemagick/src/main/java/org/alfresco/transformer/ImageMagickOptionsBuilder.java

```diff
@@ -223,7 +222,7 @@ public ImageMagickOptionsBuilder withAllowEnlargement(final String allowEnlargem
 
     public ImageMagickOptionsBuilder withAllowEnlargement(final Boolean allowEnlargement)
     {
-        this.allowEnlargement = allowEnlargement;
+        this.allowEnlargement = allowEnlargement == null ? true : allowEnlargement;
```

Comment:
If null, shouldn't it default to false ?

Reply:
Needs to be true for backward compatibility.